src/org/pentaho/reporting/platform/plugin/ExecuteReportContentHandler.java

```diff
@@ -60,9 +61,12 @@ public ExecuteReportContentHandler( final ReportContentGenerator contentGenerato
 
   public void createReportContent( final OutputStream outputStream, final Serializable fileId, final String path,
       final boolean forceDefaultOutputTarget ) throws Exception {
+    ReportListenerThreadHolder.setRequestId( contentGenerator.getInstanceId() );
+
     final long start = System.currentTimeMillis();
     final Map<String, Object> inputs = contentGenerator.createInputs();
 
+    // Deprecated call. See AuditWrapper.
```

Comment:
Can we use the new AuditWrapper here as well? I cannot see a good reason why we can't use the cleaner method everywhere. It does the same thing as before, so the behaviour is not going to change, but at least we dont end up with a mix of old and new code.